client/reducers/ats.js

```diff
@@ -0,0 +1,18 @@
+import { ATS } from '../actions/types';
+
+const initialState = {
+    ats: []
+};
```

Comment:
For my recommendation above, this should be `const initialState = []`;

Reply:
Same design decision as above: https://github.com/bocoup/aria-at-report/pull/33/files/8b0dd24941eaaf2d296a4eb46c180d335d17321f#r409082767